rate-providers/yUSDRateProviderArbitrum.md

```diff
@@ -0,0 +1,53 @@
+# Rate Provider: `ChainlinkRateProvider`
+
+## Details
+- Reviewed by: @brunoguerios
+- Checked by: @\<GitHub handle of secondary reviewer\>
+- Deployed at:
+    - [arbitrum:0xb1D5c23DbfCb36864f32bc704AE4205BB9720C33](https://arbiscan.io/address/0xb1D5c23DbfCb36864f32bc704AE4205BB9720C33)
+- Audit report(s):
+    - [Protocol Audits](https://docs.yield.fi/resources/audits)
+
+## Context
+This RateProvider is updated via the API3 product. It reports the exchangeRate of yUSD/USD on mainnet. It is read on mainnet and available as an Oracle on arbitrum.
```

Comment:
I must say I'm not able to fully grasp this API3 data source, but I'm assuming it's a standard and I replicated the findings registered on the ezETH rate provider review.
Can you guys please confirm this is indeed the case?

Reply:
Hey Bruno. Gerally speaking whenever we first took a look at a new oracle rate provider, they were reviewed as part of the review. After that if the rate is a "direct" result of a oracle price provider, we have marked it as usable. This includes oracle such as:
- API3
- Chainlink
- Redstone
- Chronicle

Reply:
In cases like this, you can refer to the review already being done at [review](./API3RateProvider.md)